lucene/core/src/java/org/apache/lucene/search/TopDocs.java

```diff
@@ -350,4 +354,38 @@ private static TopDocs mergeAux(
       return new TopFieldDocs(totalHits, hits, sort.getSort());
     }
   }
+
+  /** Reciprocal Rank Fusion method. */
+  public static TopDocs rrf(int TopN, int k, TopDocs[] hits) {
```

Comment:
nit: function arguments should use lower camel case

```suggestion
  public static TopDocs rrf(int topN, int k, TopDocs[] hits) {
```

Reply:
Oops, my bad.